src/features/class-liveblog-entry-post-type.php

```diff
@@ -0,0 +1,163 @@
+					'attributes'               => __( 'Liveblog Entry Attributes', 'wp-liveblog' ),
+					'insert_into_item'         => __( 'Insert into liveblog entry', 'wp-liveblog' ),
+					'uploaded_to_this_item'    => __( 'Uploaded to this liveblog entry', 'wp-liveblog' ),
+					'featured_image'           => __( 'Featured image', 'wp-liveblog' ),
+					'set_featured_image'       => __( 'Set featured image', 'wp-liveblog' ),
+					'remove_featured_image'    => __( 'Remove featured image', 'wp-liveblog' ),
+					'use_featured_image'       => __( 'Use as featured image', 'wp-liveblog' ),
+					'filter_items_list'        => __( 'Filter liveblog entries list', 'wp-liveblog' ),
+					'items_list_navigation'    => __( 'Liveblog Entries list navigation', 'wp-liveblog' ),
+					'items_list'               => __( 'Liveblog Entries list', 'wp-liveblog' ),
+					'item_published'           => __( 'Liveblog Entry published.', 'wp-liveblog' ),
+					'item_published_privately' => __( 'Liveblog Entry published privately.', 'wp-liveblog' ),
+					'item_reverted_to_draft'   => __( 'Liveblog Entry reverted to draft.', 'wp-liveblog' ),
+					'item_trashed'             => __( 'Liveblog Entry trashed.', 'wp-liveblog' ),
+					'item_scheduled'           => __( 'Liveblog Entry scheduled.', 'wp-liveblog' ),
+					'item_updated'             => __( 'Liveblog Entry updated.', 'wp-liveblog' ),
+					'menu_name'                => __( 'Liveblog Entries', 'wp-liveblog' ),
+				],
+				'description'      => __( 'A single update belonging to a Liveblog post.', 'wp-liveblog' ),
+				'public'           => false,
+				'hierarchical'     => true,
+				'show_in_rest'     => true,
+				'rest_base'        => 'liveblog-entries',
+				'map_meta_cap'     => true,
+				'supports'         => [
+					'title',
+					'editor',
+					'revisions',
+					'author',
+					'excerpt',
+					'custom-fields',
+				],
+				'rewrite'          => false,
+				'query_var'        => false,
+				'delete_with_user' => false,
+			],
+		);
+	}
+
+	/**
+	 * Set post type updated messages.
+	 *
+	 * The messages are as follows:
+	 *
+	 *   1 => "Post updated. {View Post}"
+	 *   2 => "Custom field updated."
+	 *   3 => "Custom field deleted."
+	 *   4 => "Post updated."
+	 *   5 => "Post restored to revision from [date]."
+	 *   6 => "Post published. {View post}"
+	 *   7 => "Post saved."
+	 *   8 => "Post submitted. {Preview post}"
+	 *   9 => "Post scheduled for: [date]. {Preview post}"
+	 *  10 => "Post draft updated. {Preview post}"
+	 *
+	 * (Via https://github.com/johnbillion/extended-cpts.)
+	 *
+	 * @param mixed[] $messages An associative array of post updated messages with post type as keys.
+	 * @return mixed[] Updated array of post updated messages.
+	 */
+	public function filter_post_updated_messages( $messages ) {
+		global $post;
+
+		if ( ! $post instanceof WP_Post ) {
+			return $messages;
+		}
+
+		$preview_url    = get_preview_post_link( $post );
```

Comment:
This is fine to merge for now but we might want to think more deeply about how to handle things like previews since entries will be part of a parent liveblog and won't be individually accessible.